.circleci/config.yml

```diff
@@ -792,6 +817,23 @@ jobs:
       <<: *infrastructure_config_environment
       PKG_TARGET_OS: ubuntu
 
+  upload_latest_packages_osx:
+    executor: infrastructure_container_stable
+    <<: *infrastructure_config
+    <<: *upload_package
```

Comment:
So this is building and uploading again:
* To GitHub (seem bad)
* To S3 (`aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/`)?

Reply:
Good catch, looks very wrong indeed.